packages/form/src/react/vertical-layout.js

```diff
@@ -22,7 +22,10 @@ const VerticalLayout = (props, context) => (
 )
 VerticalLayout.displayName = 'VerticalLayout'
 VerticalLayout.propTypes = {
-  children: PropTypes.arrayOf(PropTypes.element)
+  children: PropTypes.oneOfType([
+    PropTypes.element,
+    PropTypes.arrayOf(PropTypes.element)
+  ])
```

Comment:
@jaketrent Why not PropTypes.node on this? is it because of the inline style?

Reply:
Yes, because we operate on elements inside the component, and node is too broad to be acceptable internally.